backend/src/ml_space_lambda/app_configuration/lambda_functions.py

```diff
@@ -0,0 +1,62 @@
+#
+#   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License").
+#   You may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import json
+import logging
+
+from ml_space_lambda.data_access_objects.app_configuration import AppConfigurationDAO, AppConfigurationModel, SettingsModel
+from ml_space_lambda.utils.common_functions import api_wrapper
+
+log = logging.getLogger(__name__)
+app_configuration_dao = AppConfigurationDAO()
+
+
+@api_wrapper
+def update_configuration(event, context):
+    request = json.loads(event["body"])
+    configScope = request["configScope"]
+    version_id = request["versionId"] + 1  # increment so this will be the latest version
```

Comment:
why does it plus 1 to the requested version? If I ask for version 1, will I get version 2?

Reply:
This is updating the configuration. On the frontend they'll be modifying the current latest configuration (versionId) and so when they send a request to this lambda, that configuration will be versionId + 1.